php/class-fieldmanager-content.php

```diff
@@ -0,0 +1,60 @@
+<?php
+/**
+ * Class file for Fieldmanager_Content
+ *
+ * @package Fieldmanager
+ */
+
+/**
+ * Static content field.
+ */
+class Fieldmanager_Content extends Fieldmanager_Field {
+
+	/**
+	 * Override field_class.
+	 *
+	 * @var string
+	 */
+	public $field_class = 'content';
+
+	/**
+	 * Content to render.
+	 *
+	 * @var string
+	 */
+	public $content = '';
+
+	/**
+	 * Type of content being rendered. plaintext, html, markdown supported.
+	 *
+	 * @var string
+	 */
+	public $content_type = 'plaintext';
```

Comment:
Proposal for consideration/discussion: instead of `$content_type` and `$content`, have `$plaintext`, `$markup`, and `$markdown`. Examples:

```php
new Fieldmanager_Content( [ 'markdown' => 'Some _Markdown_' ] );
new Fieldmanager_Content( [ 'markup' => 'Some <em>Markup</em>' ] );
```

Reply:
I like this, but ended up going with @dlh01's suggestion to break out into multiple fields (I had debated that approach to begin with), `Fieldmanager_Plaintext`, `Fieldmanager_Markup`, and `Fieldmanager_Markdown`. As a result, each one uses `content` still to maintain some consistency.

```php
new Fieldmanager_Plaintext( [ 'content' => 'Some Text' ] );
new Fieldmanager_Markup( [ 'content' => 'Some <em>Markup</em>' ] );
new Fieldmanager_Markdown( [ 'content' => 'Some _Markdown_' ] );
```